nautobot/extras/forms/forms.py

```diff
@@ -951,22 +956,23 @@ class JobForm(BootstrapMixin, forms.Form):
         label="Profile job execution",
         help_text="Profiles the job execution using cProfile and outputs a report to /tmp/",
     )
-    _task_queue = forms.ChoiceField(
-        required=False,
-        help_text="The task queue to route this job to",
-        label="Task queue",
-    )
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         # Move special fields to the end of the form
-        for field in ["_task_queue", "_profile"]:
+        for field in ["_profile"]:
             value = self.fields.pop(field)
             self.fields[field] = value
```

Comment:
Maybe move the `_profile` field over to the `Job.as_form` method as well just to reduce the split-brain logic?